projects/angular/src/lib/maskito.cva.ts

```diff
@@ -0,0 +1,28 @@
+import {Directive, Input} from '@angular/core';
+import {DefaultValueAccessor, NG_VALUE_ACCESSOR} from '@angular/forms';
+import {MASKITO_DEFAULT_OPTIONS, MaskitoOptions, maskitoTransform} from '@maskito/core';
+
+@Directive({
+    selector: 'input[maskito], textarea[maskito]',
+    providers: [
+        {
+            provide: NG_VALUE_ACCESSOR,
+            multi: true,
+            useExisting: MaskitoCva,
+        },
+    ],
+    host: {
+        '(input)': '$any(this)._handleInput($event.target.value)',
+        '(blur)': 'onTouched()',
+        '(compositionstart)': '$any(this)._compositionStart()',
+        '(compositionend)': '$any(this)._compositionEnd($event.target.value)',
+    },
+})
+export class MaskitoCva extends DefaultValueAccessor {
+    @Input()
+    maskito: MaskitoOptions = MASKITO_DEFAULT_OPTIONS;
+
+    override writeValue(value: unknown): void {
+        super.writeValue(maskitoTransform(String(value ?? ''), this.maskito));
```

Comment:
Small exponential numbers will not be parsed by `String`. It can only be handled with `toFixed()`

![image](https://user-images.githubusercontent.com/26553030/224633555-d8b72e07-73ee-4d21-82e9-d4a240253dcf.png)

Reply:
That's how native `input` would also behave if you set it to its `.value`. I believe it's good enough for automatic format handling, if a person needs more control, they can always convert their values to string and parse from string on their own. As seen in this PR it is a matter of a couple of lines to create your own CVA and add custom value processing in `writeValue` and `onChange`.